packages/subgraph/scripts/deployContractsAndToken.ts

```diff
@@ -30,9 +32,37 @@ async function main() {
             from: Deployer,
         }
     );
+    // deploy native asset super token
+    await deploySuperToken(
+        (x: any) => errorHandler("SuperToken", x),
+        [":", "ETH"],
+        {
+            web3: (global as any).web3,
+            from: Deployer,
+        }
+    );
+    // deploy native super token
+    await deployNativeSuperToken(
+        (x: any) => errorHandler("NativeSuperToken", x),
+        [":", "Mr.Token", "MR", "10000000"],
+        {
+            web3: (global as any).web3,
+            from: Deployer,
+        }
+    );
+    // TODO: figure out why this script is not properly getting the token symbol
```

Comment:
You mentioned that this TODO is OK, right?

Reply:
yeah this is okay